patches/minecraft/net/minecraft/client/audio/SoundHandler.java.patch

```diff
@@ -38,3 +38,15 @@
          for (ResourceLocation resourcelocation : this.field_147697_e.func_148742_b())
          {
              SoundEventAccessor soundeventaccessor = (SoundEventAccessor)this.field_147697_e.func_82594_a(resourcelocation);
+@@ -253,6 +268,11 @@
+         this.field_147694_f.func_148615_a(p_147691_1_, p_147691_2_);
+     }
+ 
++    public void setListener(net.minecraft.entity.Entity entity, float p_147691_2_)
```

Comment:
This also needs an entry in forge.exc. Otherwise it will break upon setup.

IMHO it's better to just do the same here as for `updateRenderInfo(...)` and redirect the old method, though it's only a one liner. This way, the patch won't break anything if the implementation changes for whatever reason.

Reply:
Ok? I don't understand why. It's a new method, which means there is no binary compatibility issue.

Reply:
If you run setup again, `p_147691_2_` will be deobfuscated, but not reobfuscated again by genPatches. This means the current patch is not stable.
So you can either add the line to forge.exc which will then properly reobfuscate the parameter again, or just use the deobfuscated name right away.

Reply:
Sorry, for not responding earlier. Is there anything i have to do at the moment?

Reply:
Yes, you should either replace `p_147691_2_` with some readable name or add a line to forge.exc. Otherwise this will eventually cause some additional patch in the future when someone adds a mapping for the parameter.

As I said in the first comment, I would also just redirect the old method to the new one, as in the other patches, as it is less likely to break anything in the future. But that's just my opinion, don't know what others think about that.

Reply:
I added a line in forge.exc already, to make the patch smaller.

Reply:
I only see `SoundManager.setListener(...)` in forge.exc. This one is in `SoundHandler`.

Reply:
Is it better now?